source/developer/contribution-guide.md

```diff
@@ -66,6 +66,8 @@ After submitting a PR, before it is merged:
 3. Dev review must pass
     - Two core committers will review the pull request and either give feedback or approve the PR.
     - Any comments will need to be addressed before the pull request is ready to merge.
+4. CSS review must pass
```

Comment:
propose we move it before dev review

also, would it be in PM or Dev review stage following the contributing guidelines? https://github.com/mattermost/platform/blob/master/CONTRIBUTING.md